src/vsCodeConfig/settings.ts

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ConfigurationTarget, Uri, workspace, WorkspaceConfiguration } from "vscode";
+import { extensionPrefix } from '../constants';
+
+/**
+ * Uses extensionPrefix 'appService' unless otherwise specified
+ */
+export async function updateGlobalSetting<T = string>(section: string, value: T, prefix: string = extensionPrefix): Promise<void> {
+    const projectConfiguration: WorkspaceConfiguration = workspace.getConfiguration(prefix);
```

Comment:
I'm starting to get pretty annoyed that this setting stuff is still in an open PR ...the logic is so simple, affects so many files, is unrelated to `getDeployFsPath`, and could've easily been merged weeks ago if it was submitted as a PR by itself